### DISCUSSION TITLE
Question Pytorch Geometric and multi-GPU trianing

### DISCUSSION THREAD

**Discussion by ssnnoo** [Lightning-AI/pytorch-lightning#16686] [DDP / multi-GPU / multi-node]
I was wondering what strategies are supported for multi gpu training and pytorch geometric (ddp, fsdp_native, deepspeed etc.)? What is the most up-to-date documentation on this issue? Thank you very much!

**Comment by amorehead** [comment]
+1, since this is a somewhat common use case many new users of PyTorch Geometric may be looking for.
